clipboard_windows.py

```diff
@@ -0,0 +1,83 @@
+import ctypes
+from ctypes.wintypes import BOOL, HWND, HANDLE, HGLOBAL, UINT, LPVOID, INT, LPWSTR
+from ctypes import c_size_t as SIZE_T
+from typing import List, Tuple, Optional
+import os
+
+OpenClipboard = ctypes.windll.user32.OpenClipboard
```

Comment:
This throws an error on non-Windows systems, since it tried to build regardless of whether the option is checked.

```
AttributeError: module 'ctypes' has no attribute 'windll'
```

We can attempt to hide it under a try/except (under the principle that it's faster to try and fail than do a system lookup and use an if. hiding the option may be hard to do, but we can probably just put `(Windows only)` on it.

```suggestion
try:
    import ctypes
    from ctypes.wintypes import BOOL, HWND, HANDLE, HGLOBAL, UINT, LPVOID, INT, LPWSTR
    from ctypes import c_size_t as SIZE_T
    from typing import List, Tuple, Optional
    import os
    
    OpenClipboard = ctypes.windll.user32.OpenClipboard
    ... the rest of the code
except:
    ...something
```